roboteam_world/src/Handler.cpp

```diff
@@ -153,6 +172,33 @@ void Handler::onRobotFeedback(const rtt::RobotsFeedback& feedback) {
     std::lock_guard guard(sub_mutex);
     receivedRobotData.push_back(feedback);
 }
+void Handler::startReplay(rtt::LogFileReader& reader) {
```

Comment:
Is is possible to process the packets at the same frequency / relative timestamps they came in? I might be wrong but it looks like this code pumps all packets into observer as fast as possible. Processing the packets at the 'correct' time can help with AI playback

Reply:
Also, this will be very difficult since  differences in program scheduling by the OS may still generate completely different behavior in AI, even with if we time this observer replay to the nano-second. This is one of the reasons why I am so heavily in favor of integrating all the repositories in one application.